script.js

```diff
@@ -16,3 +16,201 @@ document.querySelectorAll('.nav_link').forEach((n) => n.addEventListener('click'
   navBranding.classList.remove('active');
   navContainer.classList.remove('active');
 }));
+
+// multistories project
+
+const multiPost = {
+  id: '_1',
+  heading: 'My Recent Works',
+  name: 'Multi-Post Stories',
+  desc: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+  featuredImg: 'image folder/works.png',
+  technologies: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+  liveLink: '#',
+  sourceLink: '#',
+};
+
+const stories = document.querySelector('.multi_stories');
+stories.innerHTML += `<div class="recent_work_div flex">
+ <h2 class="recent_work_header">${multiPost.heading}</h2>
+ <img class="image_line" src="image folder/recent_work_line.png" alt="line">
+ </div>
+
+ <div class="recent_work_content">
+ <img class="img_placeholder" src="${multiPost.featuredImg}" alt="multipost image">
+ <div class="placeholder_content">
+ <h3 class="multi_story">${multiPost.name}</h3>
+  <p class="recent_work_text">${multiPost.desc}</p>
+   <ul class="recent_work_language">
+ <li class="work_language flex">${multiPost.technologies[0]}</li>
+ <li class="work_language flex">${multiPost.technologies[1]}</li>
+ <li class="work_language flex">${multiPost.technologies[2]}</li>
+ <li class="work_language flex">${multiPost.technologies[3]}</li>
+  </ul>
+  <button data-modal-target="#modal" type="submit" class="see_prjt_buttn">See Project</button>
+ </div>`;
+
+// recent works project
```

Comment:
You did a great job creating projects and popup dynamically but there are a few thing needs to be fixed.
<img width="654" alt="Screenshot 2023-04-13 at 8 58 05 PM" src="https://user-images.githubusercontent.com/25887253/231848752-e2d0b4e7-5269-4d14-a91a-6940068f87f4.png">
<img width="1440" alt="Screenshot 2023-04-13 at 8 59 15 PM" src="https://user-images.githubusercontent.com/25887253/231848770-d0f6f107-dbdf-427a-b7b3-4185bbf383c8.png">

- [ ] A basic rule in UI design, every element needs to be aligned with another element. Programming language tags needs to be aligned with image and the project heading.
- [ ] Popup needs to be aligned to the center. Your popup has less space on the left than right.
- [ ] Popup close button needs to be aligned with the description text's right end.
- [ ] Buttons and image's bottom end needs to be aligned horizontally. Buttons aligned with the description text on the left vertically. Button text and icons needs to aligned to the center, icons are a bit higher position on yours.